src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -552,6 +650,20 @@ ov::AnyMap get_default_generate_config(const std::shared_ptr<ov::Model>& model,
     return config;
 }
 
+ov::AnyMap get_default_tail_mm_config(const std::optional<NPUDesc>& npudesc) {
+    auto config = get_default_common_config(npudesc);
+    config.erase("NPUW_SLICE_OUT");
+    config.emplace("NPUW_ONLINE_PIPELINE", "NONE");
+    if (npudesc.has_value() && npudesc->arch == "4000") {
+        config.emplace("NPU_DPU_GROUPS", 4);
+    }
+    // Specify NPUW DQ if Compiler DQ is not enabled
+    if (!npudesc.has_value() || !npudesc->compiler_dq) {
+        config.emplace("NPUW_DQ", "YES");
```

Comment:
shouldn't this be `DQ_FULL` instead?

Reply:
It's the same logic we have for prefill and generate:
common config already contains DQ and DQ full if the compiler supports it. If it's an old version which doesn't, we do NPUW DQ